styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -185,6 +185,17 @@ Zoolander.Tracking = (function Tracking($) {
       });
     });
 
+    $(document).on('click', '.solve-cta', (e) => {
+      dataLayer.push({
+        event: 'cta.click',
```

Comment:
Just realized, the `cta.click` event is incorrect. Can you change this event to `ga.event` instead?